frontend/src/components/placementVisuals/Placement.tsx

```diff
@@ -187,7 +178,7 @@ function Placement({ stepID, sampleIDs }: PlacementProps) {
                             <AddPlacementContainer onConfirm={onConfirmAddDestinationContainer} existingContainers={loadedContainers} />
                         </Col>
                         {usesSamplesheet && <Col span={3} >
-                            <Button onClick={handleGetSamplesheet} disabled={!isPlacementComplete}>Get Samplesheet</Button>
+                            <Button onClick={handleGetSamplesheet}>Get Samplesheet</Button>
```

Comment:
Just comment out the disabled... This is a temporary thing. We should put it back once it no longer cause issues ...